packages/pyroscope-flamegraph/src/convert/sandwichViewProfiles.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { treeToFlamebearer, calleesFlamebearer } from './sandwichViewProfiles';
+import { flamebearersToTree } from './flamebearersToTree';
+
+import { tree, name22FunctionTreeWithTotal } from './testData';
+
+describe('Sandwich view profiles', () => {
+  it('should return correct callees flamebearer for single function appearance', () => {
```

Comment:
I think we are missing couple other cases:

1. No data (eg no callee), not sure it will be possible (after all, how is it possible to not have a single callee?) in code but still
2. Single callee (it seems you already have this one)
3. Multiple callees

Also where are the tests for callers?

Reply:
will add more cases

Reply:
talking about `callers`......i tried to test it the same way i test `callees` and seems like [flamebearersToTree](https://github.com/pyroscope-io/pyroscope/blob/e1a7860572c944408bb92b141d3b7ce5ed06ad1d/packages/pyroscope-flamegraph/src/convert/flamebearersToTree.ts) doesnt not parse the tree node if it's total is equal to zero..i implemented callers flamegraph using empty (with `total: [0]` prop) top lvl node to accumulate values.... so `flamebearersToTree` cant return correct to test